src/App.js

```diff
@@ -1,13 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
 function App() {
+  const [todoList, setTodoList] = useState([]); 
+
+  function addTodo(newTodo) {
+    setTodoList([...todoList, newTodo]);
```

Comment:
Nothing to change here, I just wanted to add an example that mimics the example from the lesson and does the same thing. `prevTodos` can also be anything, but common patterns are to use `prevState` or `prevWhateverYourStateIs`
```suggestion
    setTodoList((prevTodos) => [...prevTodos, newTodo]);
```